wd_core.au3

```diff
@@ -839,9 +838,8 @@ Func _WD_ExecuteScript($sSession, $sScript, $sArguments = Default, $bAsync = Def
 
 	If $sArguments = Default Then $sArguments = ""
 	If $bAsync = Default Then $bAsync = False
-	If $vSubNode = Default Then $vSubNode = $_WD_JSON_RawResponse
-
-	If IsBool($vSubNode) Then $vSubNode = ($vSubNode) ? $_WD_JSON_Value : $_WD_JSON_RawResponse
+	If $vSubNode = Default Then $vSubNode = "" ; Response from web driver in JSON format
+	If IsBool($vSubNode) Then $vSubNode = ($vSubNode) ? $_WD_JSON_Value : "" ; Return JSON value node or entire response from web driver in JSON format

```

Comment:
I think this could be further defined, ie --

```
; When $vSubNode is True, configure so that the JSON value node is returned.
; When $vSubNode is False, configure so that the entire response is returned.
```

Reply:
Simler:
`; True = the JSON value node is returned , False entire JSON response is returned`
?